commit/Cargo.toml

```diff
@@ -4,8 +4,22 @@ version = "0.1.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
 
+[features]
+test-utils = ["dep:p3-dft"]
```

Comment:
I made a `p3-commit::TrivialPcs` for testing, for use in other crates. It uses `p3-dft` so I feature gated it. Not sure if there's a better way